docs/config-api.md

```diff
@@ -181,6 +181,8 @@ System.config({
 * `globals`: A map of global names to modules names that should be defined only for the execution of this module. Enables use of legacy code that expects certain globals to be present. Referenced modules automatically becomes dependencies. Only supported for the `cjs` and `global` formats. More info: [Custom Globals](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#custom-globals).
 * `loader`: Set a loader for this meta path. More info: [Plugin loaders](https://github.com/systemjs/systemjs/blob/master/docs/overview.md#plugin-loaders).
 * `sourceMap`: For plugin transpilers to set the source map of their transpilation.
+* `nonce`: The "[nonce](https://www.w3c.org/TR/CSP2/#script-src-the-nonce-attribute)" attribute of a resource. Useful for whitelisting certain resources that do need to be inlined like CJS or plugin loaders.
+* `integrity`: The "[integrity](http://www.w3.org/TR/SRI/#the-integrity-attribute)" attribute of a resource. For example, if you know the sha checksum of "src/example.js" to be ```e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855``` then you can set the ```integrity``` of a resource in the System config like ```System.config({meta: {"src/example.js": {integrity: "sha256-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"}});```
```

Comment:
Again, perhaps clarify with "The SubResource Integrity attribute for a resource"